src/BuiltInTools/dotnet-watch/HotReloadDotNetWatcher.cs

```diff
@@ -294,6 +299,13 @@ private static void ConfigureExecutable(DotNetWatchContext context, ProcessSpec
             }
         }
 
+        private string GetRelativeFilePath(string path)
+        {
+            return path.StartsWith(_workingDirectory) ?
+                $".{path.Substring(_workingDirectory.Length)}" :
```

Comment:
Is this certain to not have a trailing slash?

Reply:
You're right, we can probably make this more robust. https://github.com/dotnet/sdk/pull/23318/commits/105ee6d1a3d38d011b7c0afe95d77ade26189bab